<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>820</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MSU Latent Recognition System</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>6</y>
      <width>531</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>159</y>
      <width>1281</width>
      <height>650</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>750</y>
       <width>1661</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="status_1_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>25</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="status_2_label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="full_print">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="full_print_3">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="status_frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>610</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QLabel" name="colorbar_label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>900</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Live</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_InputLatent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Input Latent</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Original">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>400</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Original Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ROI">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>ROI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_OF">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>440</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>OF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Minutiae1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Minutiae Set 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Search">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>540</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="input_img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_LatentFname">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Minutiae2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>480</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Minutiae Set 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ViewLarger">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>View Larger</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="full_print_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="full_print_5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>330</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="full_print_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="full_print_6">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>330</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image1">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="hi!"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond1">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>290</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond3">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>290</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image3">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>70</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>600</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond5">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>599</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>379</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Correspond6">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>598</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Correspondences</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image6">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>378</y>
       <width>235</width>
       <height>215</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Up">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>280</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Down">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>330</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score1">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score3">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank3">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>350</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>350</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>350</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score5">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>350</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_score6">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>350</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_rank6">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>350</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rank: 6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname1">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname3">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>350</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname5">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>350</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fname6">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>350</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>6</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>msu_cse_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#aa00ff;&quot;&gt;MSU Latent&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#aa00ff;&quot;&gt;Recognition System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>main_frame</zorder>
   <zorder>label_2</zorder>
   <zorder>title_label</zorder>
   <zorder>lbl_name</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
